src/app/(auth)/forgot-password/page.tsx

```diff
@@ -32,39 +32,41 @@ export default function ForgotPassword() {
     setEmailSentCount(emailSentCount + 1);
   };
 
-  return emailSentCount === 0 ? (
+  return (
     <SmallCardForm onSubmit={handleForgotPassword}>
-      <H1>Forgot Password</H1>
-      <TextInput
-        label="Email"
-        placeholder="email@example.com"
-        type="email"
-        id="email"
-        errorText={emailError}
-        value={email}
-        setValue={setEmail}
-      />
-      <BigBlueButton type="submit">
-        <H4 $color="white">Send link to email</H4>
-      </BigBlueButton>
-    </SmallCardForm>
-  ) : (
-    <SmallCardForm onSubmit={handleForgotPassword}>
-      <Flex $direction="column" $gap="20px">
-        <H2>A password reset link has been sent to your email.</H2>
-        <H4 $color={COLORS.greyDark}>
-          This link will direct you to the next step. If you didn’t receive an
-          email, please click Resend Email.
-        </H4>
-        <HorizontalDiv>
+      {emailSentCount === 0 ? (
+        <>
+          <H1>Forgot Password?</H1>
+          <TextInput
+            label="Email"
+            placeholder="email@example.com"
+            type="email"
+            id="email"
+            errorText={emailError}
+            value={email}
+            setValue={setEmail}
+          />
+          <BigBlueButton type="submit">
+            <H4 $color="white">Send link to email</H4>
+          </BigBlueButton>
+        </>
+      ) : (
+        <Flex $direction="column" $align="center" $gap="20px">
+          <AuthSubHeading $align="center">
+            A password reset link has been sent to your email.
+          </AuthSubHeading>
+          <H4Centered $color={COLORS.greyDark}>
+            This link will direct you to the next step. If you didn’t receive an
+            email, please click Resend Email.
+          </H4Centered>
           <BigBlueButton type="submit">
             <H4 $color="white">Resend Email</H4>
           </BigBlueButton>
           {emailSentCount > 1 && (
-            <P $color={COLORS.greyMid}>Email has been resent!</P>
+            <P $color={COLORS.blueMid}>Email has been resent!</P>
```

Comment:
to be consistent with the verify email page, are we also planning to display the email address to which the message has been resent?

Reply:
yeah, just updated! thanks for catching this.